src/v/config/configuration.cc

```diff
@@ -1912,6 +1912,34 @@ configuration::configuration()
       "Retention in bytes for topics created during automated recovery",
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
       1_GiB)
+  , cloud_storage_recovery_topic_validation_mode(
+      *this,
+      "cloud_storage_recovery_topic_validation_mode",
+      "Validation mode performed before recovering a topic from cloud storage",
+      {.needs_restart = needs_restart::no,
+       .example = "check_manifest_existance",
+       .visibility = visibility::tunable},
+      model::recovery_validation_mode::check_manifest_existence,
+      {model::recovery_validation_mode::check_manifest_existence,
+       model::recovery_validation_mode::check_manifest_own_metadata_only,
```

Comment:
nit: what does "own" refer to?

Reply:
terrible name, changed to check_manifest_and_segment_metadata